org.bino3d.bino.yml

```diff
@@ -0,0 +1,70 @@
+app-id: org.bino3d.bino
+runtime: org.kde.Platform
+runtime-version: '6.3'
+sdk: org.kde.Sdk
+command: bino
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Play videos from the net
+  - --share=network
+  # OpenGL rendering
+  - --device=dri 
+  # Webcam capturing
+  - --device=all
+  # Audio output
+  - --socket=pulseaudio
+  # Play video files, but only read access required
+  - --filesystem=host:ro
+cleanup:
+  - '/include'
+  - '/lib/*.a'
+  - '/lib/cmake'
+  - '/lib/debug'
```

Comment:
```suggestion
```

Reply:
OK, I removed this, although I don't know why an empty /lib/debug should remain in the flatpak.

Reply:
because it's actually a separate package for the debug symbols. (`.Debug` suffix)

Reply:
OK, thanks for the explanation.